src/core_plugins/kibana/public/dev_tools/partials/dev_tools_app.html

```diff
@@ -0,0 +1,15 @@
+<div class="app-container">
```

Comment:
Then you can add another class here `<div class="dev-tools-app app-container">` and refactor your styles so that each selector has less specificity:

```less
.dev-tools-app-container {
  .flex-parent;
}

.dev-tools-container {
  .flex-parent;
}
```